src/front-end/src/components/Header/components/Navigation.tsx

```diff
@@ -18,11 +22,16 @@ const adminLinks = (
 
 const Navigation = () => {
   const { state } = useGlobalState();
+  const location = useLocation();
 
   return (
     <header>
       <nav>
-        <Link to="/search">Search</Link>
+        {location.pathname !== "/search" ? (
```

Comment:
Maybe we could just pass a prop down to SearchBar and have it render the right thing rather than doing the conditional up here? That way it's always just a SearchBar, and the behaviour can live in the component.